src/Command/UpdateSuiviTypeCommand.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace App\Command;
+
+use App\Entity\Suivi;
+use App\Manager\SuiviManager;
+use App\Repository\SuiviRepository;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+#[AsCommand(
+    name: 'app:suivi-type-update',
+    description: 'Update Suivi type when null',
+)]
+class UpdateSuiviTypeCommand extends Command
+{
+    public function __construct(private SuiviManager $suiviManager)
+    {
+        parent::__construct();
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $io = new SymfonyStyle($input, $output);
+
+        /** @var SuiviRepository $suiviRepository */
+        $suiviRepository = $this->suiviManager->getRepository();
+        $suivis = $suiviRepository->findBy([
+            'type' => 0,
```

Comment:
pourquoi `0`ça sera pas à `null` apres la migration ?

Reply:
le champ est not nullable,